### ISSUE TITLE
Insert Image dialog - Alt text input is not updated on content change

### ISSUE THREAD

**Issue by pmi** [enonic/app-contentstudio#11252]
If selected image in the dialog gets its alt text meta field updated on the server, the new value should be fetched from the server and loaded into the alt text input
